### ISSUE TITLE
can't find import: "github.com/mailgun/mailgun-go/v3/events"

### ISSUE THREAD

**Issue by CyberCatfish** [mailgun/mailgun-go#172]
The mailgun package results in an error.
can't find import: "github.com/mailgun/mailgun-go/v3/events"
A problem occurred when I go get into a newly created GAE / Go 1.9 environment.
These are projects that can be compiled without problems in the old environment.
What should I do?

> $ goenv global 1.9.4
> $ go get github.com/mailgun/mailgun-go
> $ goapp serve sample
> ...
> ERROR    2019-03-18 03:13:27,031 instance_factory.py:243] Failed to build Go application: (Executed command: /usr/local/Caskroom/google-cloud-sdk/latest/google-cloud-sdk/platform/google_appengine/goroot-1.9/bin/go-app-builder -app_base ~/Go/nack5-touch3 -api_version go1 -arch 6 -dynamic -goroot /usr/local/Caskroom/google-cloud-sdk/latest/google-cloud-sdk/platform/google_appengine/goroot-1.9 -nobuild_files ^^$ -incremental_rebuild -unsafe -binary_name _go_app -extra_imports appengine_internal/init -work_dir /var/folders/8x/zflyr_4s0x32lctg6r6brq240000gn/T/tmpZ_cqliappengine-go-bin -gcflags -I,/usr/local/Caskroom/google-cloud-sdk/latest/google-cloud-sdk/platform/google_appengine/goroot-1.9/pkg/darwin_amd64_appengine -ldflags -L,/usr/local/Caskroom/google-cloud-sdk/latest/google-cloud-sdk/platform/google_appengine/goroot-1.9/pkg/darwin_amd64_appengine main.go)
> ~/GoLib/src/github.com/mailgun/mailgun-go/events.go:8:2: can't find import: "github.com/mailgun/mailgun-go/v3/events"
> 2019/03/18 12:13:25 Can't find package "github.com/mailgun/mailgun-go/v3/events" in $GOPATH: cannot find package "github.com/mailgun/mailgun-go/v3/events" in any of:
> 	/usr/local/Caskroom/google-cloud-sdk/latest/google-cloud-sdk/platform/google_appengine/goroot-1.9/src/github.com/mailgun/mailgun-go/v3/events (from $GOROOT)
> 	~/GoLib/src/github.com/mailgun/mailgun-go/v3/events (from $GOPATH)

**Comment by thrawn01** [comment]
You will need to upgrade to go 1.11 to use mailgun-go v3. If you are unable to upgrade you can still  use mailgun-go v2 by pulling the v2.0.0 tag.

See https://cloud.google.com/appengine/docs/standard/go111/go-differences for upgrading Google App Engine to 1.11

**Comment by CyberCatfish** [comment]
Thank you, the problem is solved.

After GoGet, it seems to have caused problems with dependencies in trying to return the version in Git.
It seems that I was a little nervous to transfer many projects.
If I destroyed all Go packages and recreated, I could solve the problem.
mailgun unpacked the 2.0.0 clone zip without using GoGet.

At GAE, Go's standard is still 1.9.4, so I was worried about the trouble of package management.
Added for people who made the same mistake.
